src/lxc/string_utils.c

```diff
@@ -1028,3 +1028,135 @@ void remove_trailing_slashes(char *p)
 	while (--l >= 0 && (p[l] == '/' || p[l] == '\n'))
 		p[l] = '\0';
 }
+
+#ifndef HAVE_STRCHRNUL
+/* Taken from glibc. */
```

Comment:
For this kind of stuff we have `lxc/include/` where we track where things come from and their license.

Reply:
Ugh, I was trying to avoid that shenanigans...

Reply:
Yeah but `taken from glibc` while our licenses are compatible is a bit light on details for proper tracking of code origin :)

Reply:
Done.